packages/holocron/src/loadModule.node.js

```diff
@@ -88,7 +88,9 @@ export default async function loadModule(
       }
       nodeModule = requireFromString(moduleString, url);
     } catch (err) {
-      if (err.shouldBlockModuleReload !== false) {
+      if (process.env.NODE_ENV === 'development') {
+        console.warn('Holocron module "%s" at "%s" failed to execute.\n\t[Error Message]: "%s"\nPlease fix any errors and wait for it to be reloaded.', moduleName, url, err.message);
+      } else if (err.shouldBlockModuleReload !== false) {
```

Comment:
The benefits of `process.env.NODE_ENV === 'development'` is that this will mark the block below as dead code when `'production' === 'development'`, static analysis is done and the block conditioned by `if (err.shouldBlockModuleReload !== false) {}` will lose the `else if` and preserving the original behavior.

I think it's important that we leave authors using One App comments that address where the failure is happening.

Reply:
Do we want to add `Please fix any errors and wait for it to be reloaded or restart One App`? Is this bullet proof enough to not mention that a restart could help as well? Thankfully v5 restarts very fast.

Reply:
@mtomcal I think we can forgo this. If you have watch build running in the module that is having issues it should be picked back up.